test/NuGet.Clients.Tests/NuGet.CommandLine.Test/NuGetPackCommandTest.cs

```diff
@@ -5848,7 +5848,6 @@ public void PackCommand_ProjectFile_PackageIconUrl_WithNuspec_WithPackTask_Warns
     <OutputPath>out</OutputPath>
     <TargetFrameworkVersion>v4.0</TargetFrameworkVersion>
     <PackageIconUrl>https://test/icon.jpg</PackageIconUrl>
-    <PackageOutputPath>bin\Debug\</PackageOutputPath>
```

Comment:
Why did you need to remove this?

Reply:
That's actually the default now, as it always been, so don't need this.